src/node_process_methods.cc

```diff
@@ -454,6 +425,90 @@ static void ReallyExit(const FunctionCallbackInfo<Value>& args) {
   env->Exit(code);
 }
 
+class FastHrtime {
+ public:
+  explicit FastHrtime(std::shared_ptr<v8::BackingStore> backing_store)
+      : backing_store_(backing_store) {}
+
+  static Local<Object> New(Environment* env) {
+    v8::CFunction number_fast_func = v8::CFunction::Make(FastNumber);
+    Local<v8::FunctionTemplate> number_tmpl =
+        v8::FunctionTemplate::New(env->isolate(),
+                                  SlowNumber,
+                                  Local<Value>(),
+                                  Local<v8::Signature>(),
+                                  0,
+                                  v8::ConstructorBehavior::kThrow,
+                                  v8::SideEffectType::kHasSideEffect,
+                                  &number_fast_func);
+
+    v8::CFunction bigint_fast_func = v8::CFunction::Make(FastBigInt);
+    Local<v8::FunctionTemplate> bigint_tmpl =
+        v8::FunctionTemplate::New(env->isolate(),
+                                  SlowBigInt,
+                                  Local<Value>(),
+                                  Local<v8::Signature>(),
+                                  0,
+                                  v8::ConstructorBehavior::kThrow,
+                                  v8::SideEffectType::kHasSideEffect,
+                                  &bigint_fast_func);
+
+    Local<v8::ObjectTemplate> otmpl = v8::ObjectTemplate::New(env->isolate());
+    otmpl->SetInternalFieldCount(1);
+    otmpl->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "hrtime"), number_tmpl);
+    otmpl->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "hrtimeBigInt"),
+               bigint_tmpl);
+    Local<Object> obj = otmpl->NewInstance(env->context()).ToLocalChecked();
+
+    Local<ArrayBuffer> ab = ArrayBuffer::New(env->isolate(), 16);
+    FastHrtime* data = new FastHrtime(ab->GetBackingStore());
```

Comment:
I'm not sure where the lifetime of this should be managed.

Reply:
Probably on `IsolateData` (accessible through `env->isolate_data()`.)